### PR TITLE
update model-genomes and compartment-names

### PR DESCRIPTION
**Pull request by cnorsig**

### CODE REVIEW

File: model-genome.txt [hunk 1]
```diff
@@ -83,3 +83,31 @@ iML1515.json	pmid:29020004	ncbi_accession:NC_000913.3
...
..._003623011.1,NW_003623013.1,NW_003623028.1,NW_003623044.1,NW_003623065.1,NW_003623076.1,NW_003623089.1,NW_003623096.1,NW_003623098.1,NW_003623113.1,NW_003623117.1,NW_003623151.1,NW_003623171.1,NW_003623186.1,NW_003623206.1,NW_003623240.1,NW_003623254.1,NW_003623287.1,NW_003623294.1,NW_003623304.1,NW_003623335.1,NW_003623344.1,NW_003623345.1,NW_003623351.1,NW_003623361.1,NW_003623375.1,NW_003623380.1,NW_003623402.1,NW_003623417.1,NW_003623439.1,NW_003623445.1,NW_003623452.1,NW_003623518.1,NW_003623529.1,NW_003623532.1,NW_003623536.1,NW_003623563.1,NW_003623564.1,NW_003623572.1,NW_003623579.1,NW_003623624.1,NW_003623633.1,NW_003623653.1,NW_003623659.1,NW_003623681.1,NW_003623688.1,NW_003623695.1,NW_003623696.1,NW_003623697.1,NW_003623716.1,NW_003623738.1,NW_003623742.1,NW_003623747.1,NW_003623763.1,NW_003623771.1,NW_003623775.1,NW_003623780.1,NW_003623785.1,NW_003623809.1,NW_003623812.1,NW_003623814.1,NW_003623865.1,NW_003623904.1,NW_003623937.1,NW_003623983.1,NW_003623992.1,NW_003624014.1,NW_003624049.1,NW_003624069.1,NW_003624073.1,NW_003624080.1,NW_003624135.1,NW_003624166.1,NW_003624169.1,NW_003624174.1,NW_003624179.1,NW_003624192.1,NW_003624267.1,NW_003624269.1,NW_003624313.1,NW
```
Comment (zakandrewking) [inline model-genome.txt:87]: Is there a new iCHO model? What's this line for?

**Review by zakandrewking — CHANGES_REQUESTED** [review]
You'll also need to commit the new model files (JSON & SBML files) to the `models/` directory so I have access to them when loading the DB. Let me know if I can help with that.

File: model-genome.txt (hunk 1 above)
Comment (cnorsig) [reply model-genome.txt:87]: Yeah a new one for DG44 that reflects the loss of de novo purine synthesis so the additional req can enable selection of recomb clones

**Comment by cnorsig** [comment]
[models.zip](https://github.com/SBRG/bigg_models_data/files/3535895/models.zip)

**Comment by zakandrewking** [comment]
also need PMIDs for these

we can merge this after you merge https://github.com/cnorsig/bigg_models_data/pull/2

**Comment by cnorsig** [comment]
anything that has a PMID should have that in the file

**Pull request closed without merging**